includes/functions.php

```diff
@@ -76,13 +76,25 @@ function only_alphanumeric($string)
     return preg_replace('/[^a-zA-Z0-9]/', '', $string);
 }
 
-function logfile($string)
+function logfile($string, $level = 'error')
```

Comment:
You could use the PHP constants here instead of a string.
http://php.net/manual/en/function.syslog.php

Reply:
Do we need a none?

Reply:
Not necessarily but for us, we send our logs to /dev/null to save the disk io. I know that would be reduced if we choose info now but none seemed ok to have.

I did start to change it but imho I don't think the PHP constants fit for us. DEBUG is level 7 and INFO level 6 whilst ERR is 0. I could just skip having a debug so the check is still simple but it just doesn't seem right.

Reply:
Not sure I follow here @laf  LOG_ERR is 3.  We could definitely have the default be 2 (LOG_CRIT)

Reply:
Also have the bonus of log levels being familiar since they match syslog.

Reply:
What I mean is that the debug level is higher level then info. I.e the severity level is ascending until you get to DEBUG which then is higher than info. So we either need to skip the use of debug (we use this term in a lot of places so seems daft to drop here) or we make the check against the required level a little more complicated

Reply:
I think you are trying to make the two lists match up.  I'm saying scrap your original list in this PR and just use the syslog compatible levels.

Reply:
I don't agree personally so I don't plan on making any changes, feel free to change it if it bothers you enough.

Reply:
That's fine I just feel like you are possibly re-inventing the wheel here.  We could just use Monolog.